### ISSUE TITLE
new CSS logo?

### ISSUE THREAD

**Issue by argyleink** [CSS-Next/css-next#105]
to aid in the marketing and promotion of the new CSS era's, we're thinking it'd be nice to have a neat new logo!

the logo should:
- work well very small
- be legible
- work without color (but color def can happen)
- work without a number but also work with a number
- be able to exist as svg
- more requirements here?

this issue will hold submissions and votes, ultimately helping us find crowd sourced candidates that we can all agree on

**Comment by argyleink** [comment]
![css](https://github.com/user-attachments/assets/0238f291-fe57-4008-b49d-0514bd9a17d6)


![Frame 13](https://github.com/user-attachments/assets/05dc5cd2-acf0-48c8-a2bf-ae75b7388556)

**Comment by argyleink** [comment]
made these awhile ago https://nerdy.dev/a-css-logo-thing

![css-logo-box](https://github.com/user-attachments/assets/7e6e9233-ef77-445b-bd95-071db2e74f52)
![css-logo](https://github.com/user-attachments/assets/d79777fc-dc36-432a-909f-a0b3c46a8499)

**Comment by argyleink** [comment]
fun avatars (not a very serious submission)

![download (14)](https://github.com/user-attachments/assets/abe390c0-e9fc-467e-ac69-afbb8724a598)
![download (15)](https://github.com/user-attachments/assets/14e026fa-6a3a-4bfa-b6ca-58473d2574ba)
![download (16)](https://github.com/user-attachments/assets/ed54db69-1bc1-4eaa-8d45-13bd58749d03)

**Comment by romainmenke** [comment]
I love the idea of having a mascot for CSS.
When I was doing more projects in Go I really liked creating variations of the gopher to include in readme's. I bet the community will do fun stuff with it, if it existed.

**Comment by nileshprajapati** [comment]
I agree with @romainmenke I would like to see a mascot for CSS.

Both chameleon and Peacock look quite cool as mascot ❤️.

With regards to requirements:
- The logo needs to work in various sizes, formats and platforms (i.e. web, vs-code theme icon, etc).

Almost all icon themes with visual studies code use the `3` icon (and 5 for HTML). 

Personally, in favour of not having any number attached with the new CSS logo.

**Comment by subuhunter** [comment]
Got this Logo from My friend [Garrett](https://x.com/mcleodlabs), 


![Screenshot 2024-07-29 at 3 57 33 PM](https://github.com/user-attachments/assets/f3218971-e7c2-4100-aca4-4c7aa5f496e0)

This was his explanation for the concept he made

"here’s a quick concept. basically we keep the shield pattern that is already current. we clean up the text so it’s readable. Convert the shield to resemble a web page and add a jewel icon for the version number. "

**Comment by subuhunter** [comment]
<img width="1085" alt="Screenshot 2024-08-02 at 10 13 54 AM" src="https://github.com/user-attachments/assets/64647821-6011-44e4-9387-f486fb71170d">

Got a couple of concepts from another friend.

**Comment by enbee81** [comment]
Our design team at [9elements](https://9elements.com/) has created some logo suggestions. Here is our first proposal. While looking quite techie, it emphasizes the fluidity of CSS by incorporating a continuous line that resembles a cascade. It can be shown either horizontally or rotated by 45 degrees.

![01-01](https://github.com/user-attachments/assets/b495d25f-d637-4e2c-9559-d8b2328b9fae)
![01-02](https://github.com/user-attachments/assets/5263454f-0d90-4cf8-a8ef-e53b506852b0)
![01-03](https://github.com/user-attachments/assets/4c5b4227-0c2a-4bb2-8ae7-c59177576bd4)
![01-04](https://github.com/user-attachments/assets/57b8dad3-91c0-4d89-b094-24a8c33d2205)
![01-05](https://github.com/user-attachments/assets/0759182f-519f-42b4-9d89-ca76e58341db)

**Comment by enbee81** [comment]
For our second proposal, we aimed to creatively combine the numbers 4 and 5. During the CSS Café Meetup in Amsterdam, some concerns were raised that CSS 4 might become the forgotten version of CSS. Therefore, we thought it would be nice to showcase both versions together.

![02-01](https://github.com/user-attachments/assets/76c6c292-cae8-4b68-a067-ee277097d5ed)
![02-02](https://github.com/user-attachments/assets/e71bef94-c961-4ee6-8d8e-19bf8d09f7bd)
![02-03](https://github.com/user-attachments/assets/4765ea5d-b4ed-477e-a188-294985476833)
![02-04](https://github.com/user-attachments/assets/f6d4d4b3-1d13-4349-9117-5283f79476fa)

**Comment by enbee81** [comment]
Creating something that works at a 14px width is challenging, of course. However, we believe it is possible to use both proposed logos effectively, for example, as file type icons inside VS Code.
![vs-code](https://github.com/user-attachments/assets/de4f1564-0267-48fb-8cc9-00be955bebb1)

**Comment by romainmenke** [comment]
> this issue will hold submissions and votes

@argyleink maybe better to have a deadline and first gather submissions and only after the deadline start collecting votes?

**Comment by Amit-Sheen** [comment]
Adding my suggestion...
(The idea is to communicate with the existing JS logo, with a little nod to the CSS quirks we all love so much... 😉)

![css logo04](https://github.com/user-attachments/assets/565bc922-1cb1-41c4-b619-9cb1780a5fea)

**Comment by rol4nd909** [comment]
> Adding my suggestion...
> 
> (The idea is to communicate with the existing JS logo, with a little nod to the CSS quirks we all love so much... 😉)
> 
> 
> 
> ![css logo04](https://github.com/user-attachments/assets/565bc922-1cb1-41c4-b619-9cb1780a5fea)
> 
> 

Love this idea to keep it in the line as js 😍

**Comment by una** [comment]
Hi all! Thank you so much for the submissions so far. We just discussed some guidelines to help create constraint with the CSS-Next group:

- Recognizable at large and small sizes (i.e. in a 16px x 16px view)
- Is close to a 1:1 ratio (does not need to fit in a square, but should work nicely with other 1:1 logos)
- Limited color palette (ideally 1-2 colors so that it works on both light and dark backgrounds)
- CSS4 logo and CSS5 logo should be visually distinct

**Comment by itsjavi** [comment]
I also like consistency with the existing unofficial/official JS community logo.

Since I am a huge fan of color theory, I would chose the complementary of JS's yellow but since it's too similar to TypeScript's logo color, that would make their representation a little monotone when put together, so I would choose the pink of the Tetradic wheel.

For the font, I would use Open Sans Semi-Condensed Bold instead of Ubuntu Bold.

I would keep versioning out, because they become obsolete and tbh. does it make even sense to be talking about CSS versions nowadays? (since now every spec/module has its own separate one, e.g. CSS Color Module Level 4).

![image](https://github.com/user-attachments/assets/c222ac2b-24cf-4873-9c43-76c73f602ad9)

**Comment by enbee81** [comment]
I have to admit that I have mixed feelings about the overlapping CSS logo. On the one hand, I appreciate that it aligns nicely with the JS and TS logos. On the other hand, I wonder why CSS has to subordinate itself in terms of branding just because JS already has a logo.

Regarding the reference to the 'CSS IS AWESOME' meme, I initially chuckled, of course. However, at the same time, the meme also represents CSS as something quirky, unpredictable, and full of bugs. I’m not sure if that’s the exact message that needs to be repeated in the logo. It feels like it reinforces the recurring 'CSS is broken' mantra. To exaggerate: CSS is subordinate to JS and somehow broken.

**Comment by itsjavi** [comment]
> I have to admit that I have mixed feelings about the overlapping CSS logo. On the one hand, I appreciate that it aligns nicely with the JS and TS logos. On the other hand, I wonder why CSS has to subordinate itself in terms of branding just because JS already has a logo.
> 
> Regarding the reference to the 'CSS IS AWESOME' meme, I initially chuckled, of course. However, at the same time, the meme also represents CSS as something quirky, unpredictable, and full of bugs. I’m not sure if that’s the exact message that needs to be repeated in the logo. It feels like it reinforces the recurring 'CSS is broken' mantra. To exaggerate: CSS is subordinate to JS and somehow broken.

Not subordinate, but CSS is very related to JS since it is part of the web technologies (HTML, JS, CSS, WASM, etc.), so it makes sense to have some consistency/similarity.

It doesn't have to be exactly like that, but IMHO it's nice to have some resemblance in that direction. Also, with the future introduction of CSS functions it's kind of becoming another programming language itself.

**Comment by Muhammad-Mohsen** [comment]
A few suggestions I made yesterday.
Most of them probably wouldn't work, but maybe they give someone with more skill some ideas.
<img width="950" alt="css" src="https://github.com/user-attachments/assets/ee5dea7a-6635-4f01-ae51-fee7f80dc763">

**Comment by el22or** [comment]
Hi everyone,

I like the idea about heaving a mascot for CSS! :heart: 
Both Chameleon and Peacock look fantastic, but I would like to suggest one more awesome animal, a Mantis Shrimp! 

Mantis Shrimp has 12 cone cells and is able to see colors that we can not imagine. They can see ultra violet, ultra red and even polarized colors. They are amazing! :)

Please consider it! :shrimp:

**Comment by argyleink** [comment]
we've deliberated a few times about the logos now and have begun unanimously converging on the submission by @itsjavi:

![image](https://github.com/user-attachments/assets/746dcd70-1c07-40fd-b85c-28f303aaffb1)

justifications:
1. fits well next to other logos of it's caliber (js, ts, html, etc)
2. uses a well known CSS named color `hotpink`, a subtle and excellent hat tip to CSS
3. the border radius, also subtle and an excellent hat tip to CSS
4. works great without color (light and dark alt versions)
5. legible small or large
6. works with or without a number
7. easily can exist as svg or as css

there's so many delightful and rad submissions in here, we really appreciate the community coming together here to work on this. we felt there was clearly a winner for "radness", which is the submission by [Nine Elements](https://9elements.com/). but ultimately, we felt the submission in the rounded corner box was minimal and easily integrated with more places. thank you everyone for your submissions!

have issues with this deliberation? let us know your thoughts! 

we're reached out to Javi to create a couple more formal versions (with very minor tweaks requested), and will post them here. we'll continue sharing with the community, getting feedback, and as long as nothing major comes up, we may be looking at a next-gen CSS logo being born!

**Comment by argyleink** [comment]
![css](https://github.com/user-attachments/assets/2b4964e3-58cb-4d54-be9e-616ebf175439)

Download svg, png, light/dark versions from this zip:
[css-logo.zip](https://github.com/user-attachments/files/17063211/css-logo.zip)

we're still working on the numbered variants, but the basic icon has stabalized and this is our latest iteration 👍🏻

**Comment by romainmenke** [comment]
Maybe interesting to "open source" figma, illustrator, ... files?
Then people can make subtle variants, or add subtitles?

**Comment by argyleink** [comment]
yep, will do!
we've also got [a working Codepen](https://codepen.io/argyleink/pen/dyBxoEr/43ecbcaee79290bc96dfe543bedb7213) (but uses text-box-trim, so needs Canary or Safari TP)
will share more official files and pens once all the dust has settled 

<img width="1437" alt="Screenshot 2024-09-19 at 10 13 08 AM" src="https://github.com/user-attachments/assets/9ac7d646-3911-4aa9-80b1-ecaf5fda7ccc">

**Comment by argyleink** [comment]
please hold off on using these logos until we've resolved some issues

**Comment by romainmenke** [comment]
Was also going to wait until there is an official reveal.
Don't want to steal your thunder ;)

**Comment by chriskirknielsen** [comment]
While I was more of the opinion of having a non-JS-adjacent logo, I still think this is a good candidate, though I have some notes!

The text size seems a little too small to me. Since Web Assembly (that's what WA is, right? 😅) uses a different font, I think it's reasonable to use a more appropriate font here as well. Javi's original design seems closer to what I'd have in mind, instead of "forcing" the use of the Neutra font used for JS.

![A CSS logo on a pink square with rounded corners besides top left. A text 'CSS' sits on the bottom-right using a narrower font than the current candidate logo.](https://github.com/user-attachments/assets/9eb55cc6-bd27-4389-b218-f717ab6ee1ab)
(note that I opened up the aperture of the C so it would line up with the terminal of the S)

By using a narrower font ([open source DINish](https://github.com/playbeing/dinish) Condensed Heavy in this example but any condensed font could be accepted), we can get back to the same character height in the square (compared to JS/TS). This also increases the logo readability at smaller sizes:

<img width="140" height="50" alt="Initial CSS logo change at small size" src="https://github.com/user-attachments/assets/2ec93ce2-c370-4bd7-a8a4-93fb6d2a5cc7">
<img width="140" height="50" alt="Proposed CSS logo change at small size" src="https://github.com/user-attachments/assets/8f922661-2245-488d-ab7f-2a90de542502">

It keeps the logo looking like it's part of the same family, but ensures it has its own look and feel, instead of being that sibling of JavaScript who's always trying to copy it. 😄

**Comment by enbee81** [comment]
I totally understand the reasoning for choosing a logo that plays nicely with the existing JS and other logos.
I'd also go with [chriskirknielsen](https://github.com/chriskirknielsen)'s suggestion to make the letters a bit bigger to increase readability. Even though you seem to have already decided on the final logo, I wanted to suggest one last option: a combination of one of the logos we created with the now-chosen partly rounded hot-pink square.

<img width="1476" alt="css" src="https://github.com/user-attachments/assets/ece82fec-606c-41ca-b584-cddb6aacc7f0">

I especially like the border radius that is matched in the bottom curve of the "C" and the second "S".

**Comment by bramus** [comment]
+1 on making the text size bigger, as [it was here](https://github.com/CSS-Next/css-next/issues/105#issuecomment-2287469583). It was the first thing that I noticed too when seeing them side-by-side.

A pity that [the overflow pun](https://github.com/CSS-Next/css-next/issues/105#issuecomment-2269099876) was dropped – it’s what makes CSS, CSS – but I guess that’s OK for alignment purposes.

**Comment by argyleink** [comment]
> By using a narrower font ([open source DINish](https://github.com/playbeing/dinish) Condensed Heavy in this example but any condensed font could be accepted), we can get back to the same character height in the square (compared to JS/TS). This also increases the logo readability at smaller sizes:

here's the design using DINish

<img width="847" alt="Screenshot 2024-09-23 at 9 02 37 AM" src="https://github.com/user-attachments/assets/ccf5c99c-48b1-42cd-a6b7-fd6bc4e96cab">


nice condensed sans-serif indeed.

**Comment by subuhunter** [comment]
Wow, this looks even better with the new font!!! 😍

**Comment by itsjavi** [comment]
Good suggestions guys, I like both @enbee81 's and DINish fonts, but the latter is easier to represent in CSS as it exists as a font already.

I also agree we don't have to use the same font as the other web language logos.

@argyleink your changes look good to me :)

**Comment by Que-tin** [comment]
> > By using a narrower font ([open source DINish](https://github.com/playbeing/dinish) Condensed Heavy in this example but any condensed font could be accepted), we can get back to the same character height in the square (compared to JS/TS). This also increases the logo readability at smaller sizes:
> 
> here's the design using DINish
> 
> <img alt="Screenshot 2024-09-23 at 9 02 37 AM" width="847" src="https://private-user-images.githubusercontent.com/1134620/369956475-ccf5c99c-48b1-42cd-a6b7-fd6bc4e96cab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxMTUzODcsIm5iZiI6MTcyNzExNTA4NywicGF0aCI6Ii8xMTM0NjIwLzM2OTk1NjQ3NS1jY2Y1Yzk5Yy00OGIxLTQyY2QtYTZiNy1mZDZiYzRlOTZjYWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjNUMTgxMTI3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWNkNjZmNWI1NmIxOTY2YjdhNmVhMzM1MjFiYzE4NmFlOWU4NWYxYmUxOTY1NzFmNDJmMGE4YmQxYzgxYjY0YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.DNDqarTd-HupVlg7zXGGIeirlevWd531FYQ36EqtGpU">
> nice condensed sans-serif indeed.

@argyleink Do you have a comparison between JS, TS, WASM and ours at hand? So one can see the font-size difference with the DIN font? Looks really good tho, especially the tray icons are way more readable.

**Comment by Que-tin** [comment]
If it would just be a little lighter I think it would be perfect imo. What's the font-weight black?

**Comment by argyleink** [comment]
correct, i havent updated the TS one to use Segoe yet. if we're going for super correct, the JS and TS letters are much larger in most cases. currently normalized in my codepen.

DINish is a variable font, we could reduce the weight if needed 👍🏻

**Comment by Que-tin** [comment]
> +1 on making the text size bigger, as [it was here](https://github.com/CSS-Next/css-next/issues/105#issuecomment-2287469583). It was the first thing that I noticed too when seeing them side-by-side.
> 
> A pity that [the overflow pun](https://github.com/CSS-Next/css-next/issues/105#issuecomment-2269099876) was dropped – it’s what makes CSS, CSS – but I guess that’s OK for alignment purposes.

We actually discuessed about the overflow pun for a while and agrred upon that the CSS logo shouldn't be based on a meme. I mean it its funny, but can it be widely taken serious this way?
Also it's pretty hard use these icons as trayicons, as they most of them time aren't in a 1:1 format and therefore will be scaled down and therefore less readable.

**Comment by rol4nd909** [comment]
> <img width="855" alt="Screenshot 2024-09-23 at 11 25 17 AM" src="https://github.com/user-attachments/assets/093dea10-a21f-4b2e-b5ba-bcfd0e2d19b8">
> 
> 

This looks so nice! Awesome suggestion for the new font

**Comment by romainmenke** [comment]
It is not uncommon to have circular "profile" pictures on various platforms.
Should there be a variant logo where the text is centered horizontally and vertically?


For example on GitHub with `WA`:
<img width="191" alt="Screenshot 2024-09-24 at 13 19 40" src="https://github.com/user-attachments/assets/84c080d8-076b-4aad-ae86-41e416ef08d8">

**Comment by villepreux** [comment]
Am I the only one that finds that black on pink (`#000000` on `#FF69B4`) less readable (in terms of contrast) than, say, white on (a darker) pink (for example `#FFFFFF` on `#DB0071`) ?

I know `#000000` on `#FF69B4` passes WCAG AAA but we know that algorithm has flaws and don't take font size into account. OTOH `#FFFFFF` on `#DB0071` passes WCAG AA and APCA and is IMHO (and for my eyes) way much more readable. 

But maybe that's just me and my eyes...
Great work anyway

**Comment by rol4nd909** [comment]
The latest version we are now working on has these 3 variations

Main
This is the main logo colorway. It uses pinkish #E5007C as background color and white #FFFFFF as font color.

Dark
The dark colorway of the logo. It uses black #000000 as background color and white #FFFFFF as font color. This logo should only be used on light backgrounds to ensure accessibility.

Light
The light colorway of the logo. It uses white #FFFFFF as background color and black #000000 as font color. This logo should only be used on dark background to ensure accessibility.

**Comment by rol4nd909** [comment]
> Am I the only one that finds that black on pink (`#000000` on `#FF69B4`) less readable (in terms of contrast) than, say, white on (a darker) pink (for example `#FFFFFF` on `#DB0071`) ?
> 
> I know `#000000` on `#FF69B4` passes WCAG AAA but we know that algorithm has flaws and don't take font size into account. OTOH `#FFFFFF` on `#DB0071` passes WCAG AA and APCA and is IMHO (and for my eyes) way much more readable.
> 
> But maybe that's just me and my eyes... Great work anyway

The latest version we are now working on has these 3 variations

Main
This is the main logo colorway. It uses pinkish #E5007C as background color and white #FFFFFF as font color.

Dark
The dark colorway of the logo. It uses black #000000 as background color and white #FFFFFF as font color. This logo should only be used on light backgrounds to ensure accessibility.

Light
The light colorway of the logo. It uses white #FFFFFF as background color and black #000000 as font color. This logo should only be used on dark background to ensure accessibility.

**Comment by villepreux** [comment]
> > Am I the only one that finds that black on pink (`#000000` on `#FF69B4`) less readable (in terms of contrast) than, say, white on (a darker) pink (for example `#FFFFFF` on `#DB0071`) ?
> > I know `#000000` on `#FF69B4` passes WCAG AAA but we know that algorithm has flaws and don't take font size into account. OTOH `#FFFFFF` on `#DB0071` passes WCAG AA and APCA and is IMHO (and for my eyes) way much more readable.
> > But maybe that's just me and my eyes... Great work anyway
> 
> The latest version we are now working on has these 3 variations
> 
> Main This is the main logo colorway. It uses pinkish `#E5007C` as background color and white `#FFFFFF` as font color.
> 
> Dark The dark colorway of the logo. It uses black `#000000` as background color and white `#FFFFFF` as font color. This logo should only be used on light backgrounds to ensure accessibility.
> 
> Light The light colorway of the logo. It uses white `#FFFFFF` as background color and black `#000000` as font color. This logo should only be used on dark background to ensure accessibility.

Oh **great** !I missed that. That's **much** better !
(IMHO a very slightly darker pink like `#E00079` would be more readable when used with a small font size though ;) but maybe less vibrant)

**Comment by oscarotero** [comment]
It would be great if the CSS logo could be reproduced easily with only CSS.
I've created this logo (for a different project some days ago) but I'd be happily donate it if you like as the CSS logo.
<img width="285" alt="imaxe" src="https://github.com/user-attachments/assets/bd956a62-02fb-4ad6-94e3-452054156ba1">

Here you can see the logo reproduced with only HTML + CSS: https://codepen.io/oscarotero/pen/MWNjeXq

**Comment by olets** [comment]
> this issue will hold submissions and votes, ultimately helping us find crowd sourced candidates that we can all agree on

As of this writing, https://github.com/CSS-Next/css-next/issues/105#issuecomment-2269099876 has the most votes by far.

**Comment by davorsuljic** [comment]
I think that CSS logo could be created with some "CSS font" - to connect technology with visual presentation. 

Just as an idea, here is my suggestion (based on "CSS font" that I created some time ago - [https://davorsuljic.github.io/css-border-font](https://davorsuljic.github.io/css-border-font)). 

![css logo light](https://github.com/user-attachments/assets/2f8fa59b-ace7-4154-a2ad-326c25f1a2b6) ![css logo dark](https://github.com/user-attachments/assets/34e2fe0b-672a-4dcb-be07-6169a8afdc41)

There are some other "CSS fonts" on codepen or on [https://a.singlediv.com/](https://a.singlediv.com/) , for example.

**Comment by el22or** [comment]
I believe the idea of creating a CSS logo using a 'CSS font' is a creative way to link the technology with its visual representation. However, we could ask the same question for other languages like JavaScript or TypeScript – should their logos also be made with their own fonts? Ultimately, it's just a logo, and while all ideas are valuable, no design will ever please everyone. Let's finalize the logo for now, and anyone who feels inspired to improve it in the future is more than welcome to share their thoughts or propose new designs. We can always revisit it when a better version is ready.

**Comment by usqr** [comment]
Here is my design

<table><tbody><tr><td colspan="2"><figure class="image"><img src="https://github.com/user-attachments/assets/e71e7384-6bb1-46e5-8db1-db36973ed78a"></figure></td></tr><tr><td><figure class="image"><img src="https://github.com/user-attachments/assets/e944ab56-aa10-49ba-8f76-22ee68d5544f"></figure></td><td><figure class="image"><img src="https://github.com/user-attachments/assets/d2aa06d8-3d76-4a2e-a259-43d88dcfdba5"></figure></td></tr><tr><td><figure class="image"><img src="https://github.com/user-attachments/assets/e99cc9c9-a5a5-4790-8749-b13f2faed4f8"></figure></td><td><figure class="image"><img src="https://github.com/user-attachments/assets/63005b4d-d71a-4f13-969a-739ecbd23677"></figure></td></tr><tr><td><figure class="image"><img src="https://github.com/user-attachments/assets/65c0ab6c-c330-43a3-9bf3-d9f4e8e0fb59"></figure></td><td><figure class="image"><img src="https://github.com/user-attachments/assets/cbd065a2-c368-47af-a1a8-ec30e4900a17"></figure></td></tr><tr><td><figure class="image"><img src="https://github.com/user-attachments/assets/6f196681-56c7-4ca2-a29b-29e5b262e394"></figure></td><td><figure class="image"><img src="https://github.com/user-attachments/assets/dad74779-5f01-4fa3-8a05-338004706d69"></figure></td></tr><tr><td colspan="2"><figure class="image"><img src="https://github.com/user-attachments/assets/6b7c626f-4955-4eeb-9f80-96ed74a78ef1"></figure></td></tr></tbody></table>

**Comment by aaaaargZombies** [comment]
> ![image](https://private-user-images.githubusercontent.com/122741/357621978-c222ac2b-24cf-4873-9c43-76c73f602ad9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NTUwMzMsIm5iZiI6MTcyODU1NDczMywicGF0aCI6Ii8xMjI3NDEvMzU3NjIxOTc4LWMyMjJhYzJiLTI0Y2YtNDg3My05YzQzLTc2YzczZjYwMmFkOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMFQxMDA1MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMjY1NjU3MGNkNTE0ZjdmNWE0OGYyNzgxMzIwYjAwNGUzMjIwOTk2MWQ5MTczYzg2N2I4ZWZkNjFlMGNmOTA5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KbC2gq_c18-unH5V9dCMCaCnheLDGsSCxXUZ38Un4b8)

This peeking out `s` is amazing, it would make me so happy if it became the logo

**Comment by mediaformat** [comment]
Which ever logo is selected, IMHO it should be reproducible via CSS

**Comment by nclm** [comment]
Hello, just finding this now, so wanted to share a quick draft I had made a few years ago for a new CSS / HTML / JS logo idea, each based on their more iconic bracket styles:

![Old icon concept for HTML, CSS and JS, each with punctuation-inspired shapes: red angle brackets for HTML, blue curly brackets for CSS, yellow round brackets for JS.](https://github.com/user-attachments/assets/3e093b45-b9b0-4f61-8bae-c78f1a62687c)

A lot to tweak for it to work well, but wanted to share that direction :)

**Comment by argyleink** [comment]
> Hello, just finding this now, so wanted to share a quick draft I had made a few years ago for a new CSS / HTML / JS logo idea, each based on their more iconic bracket styles:

great submission, the direction and connection to their dominant braces is nice and meaningful. thanks for droppin that here!

**Comment by davorsuljic** [comment]
Second idea - [https://codepen.io/davorsuljic/pen/QWedmyb](https://codepen.io/davorsuljic/pen/QWedmyb) - based on another "css font" - []([https://davorsuljic.github.io/css-fluid-grid-font](https://davorsuljic.github.io/css-fluid-grid-font).)


![css logo2 dark](https://github.com/user-attachments/assets/a819e3a8-7680-45a0-b03f-959053ca67a4)

**Comment by enbee81** [comment]
> Most of the logos I see here don’t seem to make any reference to the Cascade, which is the main reason CSS exists under that name. [usqr](https://github.com/usqr) seems to have taken it into consideration, which is nice, but I’m personally not a fan of the proposal.

If you scroll up a bit we did think about the cascade. One single fluid line cascading from one s to the next:
https://github.com/CSS-Next/css-next/issues/105#issuecomment-2264753885

**Comment by mweitzel** [comment]
Hey everyone, here's a couple of options I thought would be really cool. I think it is really important that CSS be a welcoming community like it has to me, so I have two high resolution images. They show off both "CSS" as a logo's main emblem, but also show that it can be "cool!" in a fun and quirky way. I wanted to show off techniques you can achieve with css, like stretching to fit a square and transforms. The first one is 1080x1080, since logos are square, but the second is a little off because I took a screenshot since I couldn't get the channels to export right (I made them in Gimp instead of CSS I hope that is ok). While the first one is cool, and probably more acceptable in an enterprise environment, the second one is really cool because it doesn't use all of the colors, and also helps us remember that not everyone can see all the colors. Not to mention that the "green flash" at sunset is a fabled experience only few can achieve, so it really ties into mythology a bit too.

The first one is called "Untitled 2 png final.jpeg" and the second is "Screenshot_20241016_101638.png"

![Untitled 2 png final](https://github.com/user-attachments/assets/0a2fd5ea-8bc3-400c-9932-293d2ddc0444)
<img width="539" alt="Screenshot_20241016_101638" src="https://github.com/user-attachments/assets/910b31cd-5458-4d99-8e91-e1eba34f5a52">

**Comment by itsjavi** [comment]
what are your thoughts about using a "cascade / animation" effect like this? don't pay much attention to the color change (we are just trying different ones like `rebeccapurple`).

![image](https://github.com/user-attachments/assets/a00bd2b7-2337-4eec-94ea-aa5b876e4c5d)

**Comment by rol4nd909** [comment]
The first thing was it gave me a dizzy feeling. I need to hold me phone still to not get that feeling. Besides that I must say I like the idea. But I wonder if I'm the only one with the dizzy feeling.

**Comment by Que-tin** [comment]
> The first thing was it gave me a dizzy feeling. I need to hold me phone still to not get that feeling. Besides that I must say I like the idea. But I wonder if I'm the only one with the dizzy feeling.

I actually feel the same and I normally don't have any kind of motion sickness etc.. I think the logo couldn't be used due to accessibility issues.

**Comment by chriskirknielsen** [comment]
Doing a little catch-up here, forgive my multi-tagging:

@oscarotero I certainly like [a geometric CSS logo](https://chriskirknielsen.com/projects/devographics-surveys/#state-of-css) but it can be hard to read and parse as "CSS", in my opinion. Cool that it can easily be made in CSS though!

@nclm I remember seeing these a while ago (very randomly, no idea how I came across them), it's pretty sweet. 😃

@itsjavi I get the idea but it seems like it might be a little too busy and hard to read? I don't know if I'd call it dizzying but yeah when you try to read one letter, the other behind competes a little for attention so it feels weird, if that makes sense?

@ManzDev I like the idea, it kinda pays homage to the CSS3 logo! Almost negative space, very cool exploration. Though I think since this is a re-branding of sorts, I'm not opposed to `deeppink`, and I could also go for some `rebeccapurple` like Javi suggested! Riffing off of your idea with the DINish font (I'm not married to it but I do like it a lot!), I think the number would need a better contrast to remain legible, my understanding being that the number would actually need to stand out:
<img width="250" alt="A pink CSS logo with a slightly lighter pink background inset on the right half" src="https://github.com/user-attachments/assets/374b3eca-aa8b-4441-a5a0-01b31a740ac6"> <img width="250" alt="A pink CSS logo with a 5 in the background" src="https://github.com/user-attachments/assets/c227277c-0a15-471f-a86c-99d36a45f195"> <img width="250" alt="A purple CSS logo with a 55 in the background" src="https://github.com/user-attachments/assets/51d3ecc2-d8c4-408f-8fe0-d55102e0f011">
(using 55 to test longevity 😄)

Either way it's cool to see all these ideas here! 😊

**Comment by itsjavi** [comment]
Thank you for all your feedback and suggestions. I also had doubts about accessibility and the motion effect.

Incorporating some of your ideas, and giving another try to the concept of "cascade" and dynamism as a part of CSS, here are some drafts:

A)
![image](https://github.com/user-attachments/assets/f13aead8-5b92-4040-ab10-ae4ff60127ef)

B)
![image](https://github.com/user-attachments/assets/6d86bbd0-afd5-4822-975f-754d360450a1)

The lines are just decorative, and the inner rectangle takes now just 1/4 of the logo, so the version is more clearly an independent element.

**Comment by keithjgrant** [comment]
@chriskirknielsen I really like this one. I think the negative space/coloring is the by far my favorite approach to the numbering so far

**Comment by davorsuljic** [comment]
With neo-brutalism style:

![first-version-css-logo - CSS in a square with black drop shadow. Second S in "CSS" is also black and partially hidden. There is text overflow outside the box.](https://github.com/user-attachments/assets/65555bca-70f2-4d71-a8a4-3d24e14a816b) 

![second-version-css-logo - Similar to the first version. All letters are within the box now. First S in CSS is transparent with black text stroke, places in the middle and on top of other two letter. Second S is translated down and has identical dark lime green color as C.](https://github.com/user-attachments/assets/5a3db8da-83ac-409b-a32c-b46a1ab3a3a9)


Work in progress - [https://codepen.io/davorsuljic/pen/wvVeVJw](https://codepen.io/davorsuljic/pen/wvVeVJw)

**Comment by alvaromontoro** [comment]
> <img width="855" alt="Screenshot 2024-09-23 at 11 25 17 AM" src="https://github.com/user-attachments/assets/093dea10-a21f-4b2e-b5ba-bcfd0e2d19b8">
> 

This looks nice. Although I would prefer the fonts to be more consistent. At least size-wise (and the same family too?)

**Comment by alvaromontoro** [comment]
The colors and shapes look really nice and are meaningful. My comment was about the font sizes. While the fonts are not the same, once the logos are put together like this, the height difference is noticeable and _weird_: JS and TS have the same height, CSS height is slightly larger, and WA is considerably smaller (maybe to avoid occupying too much horizontally?)

![Icons for CSS, JS, TS, and WA. A red line is drawn showcasing the height differences (CSS is slightly taller, WA is considerably shorter)](https://github.com/user-attachments/assets/39d6c918-5310-459e-855d-8200e9fdda8e)

I would have expected to have the letters have similar height. Something like this (resized CSS and WA to match JS/TS):

![Icons for CSS, JS, TS, and WA. The letters have the same height in all of them](https://github.com/user-attachments/assets/80775f42-6a47-4935-9402-c202b5af0c32)

**Comment by argyleink** [comment]
that's just in the prototype "with friends". but if you go check out the sizes of the logos in the wild, i dont think there's as much height normalization?

**Comment by SakuHeli0s** [comment]
I'm not sure if something has already been chosen, and... well, I'm new around here, I still have a lot to learn, and I saw this thread on Twitter and thought if I could share my design idea. To be honest, I'm a bit embarrassed, but I hope you like it or that it can help in some way.

![Captura de pantalla 2024-10-22 222849](https://github.com/user-attachments/assets/2a9ae5e1-f0db-4b27-8eb8-c93238e2b99c)
![Captura de pantalla 2024-10-22 222928](https://github.com/user-attachments/assets/943ab1a7-9672-4761-aae8-99ff07d8d34c)
![Captura de pantalla 2024-10-22 223007](https://github.com/user-attachments/assets/385a781c-cfa0-4543-b9b1-ee9352ed8290)

I applied a bit of the cascade idea that some designers proposed earlier, but I kept the blue since I've always seen that CSS is more known for its blue color and I left the white as well in the font. Additionally, I added the braces, which are used the most in CSS. I did it in Photoshop to give a general idea; sorry if I didn't do it in CSS.

**Comment by DomiDex** [comment]
Here is  one based on the box model. a box within a box 
![Group 1](https://github.com/user-attachments/assets/773e4024-f258-4c9b-8265-25b7da3303ef)
![Group 4](https://github.com/user-attachments/assets/714ece2b-b029-4821-9b57-307a154ac399)

**Comment by badcat** [comment]
So late to this party...

Or so very early (5 years ago)
![webcore4](https://github.com/user-attachments/assets/c2044bac-7cde-4ec4-a254-38c1d10b65f2)
https://github.com/badcat/webcore4  oh well

**Comment by MarcelHekkert** [comment]
> Hello, just finding this now, so wanted to share a quick draft I had made a few years ago for a new CSS / HTML / JS logo idea, each based on their more iconic bracket styles:
> 
> ![Old icon concept for HTML, CSS and JS, each with punctuation-inspired shapes: red angle brackets for HTML, blue curly brackets for CSS, yellow round brackets for JS.](https://private-user-images.githubusercontent.com/923008/375678337-3e093b45-b9b0-4f61-8bae-c78f1a62687c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4OTU4MzYsIm5iZiI6MTcyOTg5NTUzNiwicGF0aCI6Ii85MjMwMDgvMzc1Njc4MzM3LTNlMDkzYjQ1LWI5YjAtNGY2MS04YmFlLWM3OGYxYTYyNjg3Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNVQyMjMyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NTU2MDg0MWY0YThhYzI3Mzg4NzA0ODFhZTEyMTVkMmE3OGZmZGM2ZWE4ZmUzNGY2NWViNDE3NWViNGZjMjIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._LQ80XfRR8amUr8y_pGU5mmE7h4p_YHo7M1uSg9oCCA)
> 
> A lot to tweak for it to work well, but wanted to share that direction :)
> 
> **Edit:** AI source file if anyone wants to play with it: [openlogos_1.zip](https://github.com/user-attachments/files/17351285/openlogos_1.zip)

I like this one a lot. It references the languages by their language specific elements. Well done!

**Comment by argyleink** [comment]
## time to vote!

We feel there's been a healthy amount of iteration, creativity, and identified requirements that we're excited to propose 2 champion variants and let the community pick which is best. 

> We'll continue ideating on the variant with the number in a subsequent issue. 

See these [RC4 logo candidates in a Codepen](https://codepen.io/argyleink/pen/jOgZpmy)

<br><br>

## `rebeccapurple`

This color has <a href="https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/">meaningful history</a> to CSS, is a keyword easily used in css <code>rebeccapurple</code> and has great contrast with the bright light letters on it.

<img width="250" src="https://github.com/user-attachments/assets/58742537-7c42-4d84-bcdc-54db8cb59aa8"/>

<img width="1209" alt="Screenshot 2024-10-29 at 2 02 46 PM" src="https://github.com/user-attachments/assets/a030c6fb-ff28-48dc-9957-c01927a5fb44">

<br><br>

## Blue from the CSS3 shield

CSS original logos, the W3C logos, and the CSS3 shield have all been blue, or a tint/shade of it. It's recognizable, unoriginal, and has good contrast.

However, there are multiple blue potentials ([try them in this RC4 codepen](https://codepen.io/argyleink/pen/jOgZpmy)). Spec standard link blue, blue from the W3C logo, and blue from the CSS3 shield (the light blue from the shield does not provide enough contrast). Shown here is the shield blue since it's the most vibrant and likely most recognizable blue associated with a CSS logo.

<img width="250" src="https://github.com/user-attachments/assets/a4960db4-27a5-4bd6-a7be-11b9ed799dd3"/>

<img width="1209" alt="Screenshot 2024-10-29 at 2 03 39 PM" src="https://github.com/user-attachments/assets/1a3a490c-4441-4570-be18-479eeed03d18">

<br><br>

Reply with 🎉 to vote for the `rebeccapurple` logo  
Reply with 🚀  to vote for the `blue` logo

**Comment by DomiDex** [comment]
🎉The colour is nice. But if you want to keep the original colour, then the
blue solution

**Comment by michael-andreuzza** [comment]
> I totally understand the reasoning for choosing a logo that plays nicely with the existing JS and other logos. I'd also go with [chriskirknielsen](https://github.com/chriskirknielsen)'s suggestion to make the letters a bit bigger to increase readability. Even though you seem to have already decided on the final logo, I wanted to suggest one last option: a combination of one of the logos we created with the now-chosen partly rounded hot-pink square.
> 
> <img alt="css" width="1476" src="https://private-user-images.githubusercontent.com/29651426/369866648-ece82fec-606c-41ca-b584-cddb6aacc7f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyODA1MTAsIm5iZiI6MTczMDI4MDIxMCwicGF0aCI6Ii8yOTY1MTQyNi8zNjk4NjY2NDgtZWNlODJmZWMtNjA2Yy00MWNhLWI1ODQtY2RkYjZhYWNjN2YwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMwVDA5MjMzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmZWRkYzVhMjNiZjdlOTA5ODMxMWUzNzIwYjM2MTQ1MzkyNzY2ZjhmZWVhMWFlZDNjMzllNTZkODFhM2JiNjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Xqk9Ox3D4bcIYwCWmfCwRCWCcfRzCIBVPZ89QngTcf4">
> I especially like the border radius that is matched in the bottom curve of the "C" and the second "S".

The fact that the letters are sliced decreases readability IHMO...

**Comment by michael-andreuzza** [comment]
> ## time to vote!
> We feel there's been a healthy amount of iteration, creativity, and identified requirements that we're excited to propose 2 champion variants and let the community pick which is best.
> 
> > We'll continue ideating on the variant with the number in a subsequent issue.
> 
> See these [RC4 logo candidates in a Codepen](https://codepen.io/argyleink/pen/jOgZpmy)
> 
> ## `rebeccapurple`
> This color has [meaningful history](https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/) to CSS, is a keyword easily used in css `rebeccapurple` and has great contrast with the bright light letters on it.
> 
> <img alt="" width="250" src="https://private-user-images.githubusercontent.com/1134620/381304018-58742537-7c42-4d84-bcdc-54db8cb59aa8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyODA0MTQsIm5iZiI6MTczMDI4MDExNCwicGF0aCI6Ii8xMTM0NjIwLzM4MTMwNDAxOC01ODc0MjUzNy03YzQyLTRkODQtYmNkYy01NGRiOGNiNTlhYTgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMzBUMDkyMTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmEyNDczMTBlNWY5ODAwZmE0MjlhNmM1MmU0YjUyYWEyNTM3ODVjOTVkODZhMDA2NzNjZTBiMDkyMjY3MWJhOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.OIUKoK6YFUK9MKTXgEbHMvCqi_tJxi1n4_vkk5K6KJ8"> <img alt="Screenshot 2024-10-29 at 2 02 46 PM" width="1209" src="https://private-user-images.githubusercontent.com/1134620/381306182-a030c6fb-ff28-48dc-9957-c01927a5fb44.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyODA0MTQsIm5iZiI6MTczMDI4MDExNCwicGF0aCI6Ii8xMTM0NjIwLzM4MTMwNjE4Mi1hMDMwYzZmYi1mZjI4LTQ4ZGMtOTk1Ny1jMDE5MjdhNWZiNDQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMzBUMDkyMTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTY4MjhlODhiOGVlODVkNzIxNmM5ZjA1NzE3YjAzODk2ODMyMTRhMmMzZmY5NGJkNzlkZGRjYjMxN2JiNzU1ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.S7aZ5uG0HCsNSg1FZLMuLSLU7xeXHWIJ8e3dynfiC3c">
> ## Blue from the CSS3 shield
> CSS original logos, the W3C logos, and the CSS3 shield have all been blue, or a tint/shade of it. It's recognizable, unoriginal, and has good contrast.
> 
> However, there are multiple blue potentials ([try them in this RC4 codepen](https://codepen.io/argyleink/pen/jOgZpmy)). Spec standard link blue, blue from the W3C logo, and blue from the CSS3 shield (the light blue from the shield does not provide enough contrast). Shown here is the shield blue since it's the most vibrant and likely most recognizable blue associated with a CSS logo.
> 
> <img alt="" width="250" src="https://private-user-images.githubusercontent.com/1134620/381304014-a4960db4-27a5-4bd6-a7be-11b9ed799dd3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyODA0MTQsIm5iZiI6MTczMDI4MDExNCwicGF0aCI6Ii8xMTM0NjIwLzM4MTMwNDAxNC1hNDk2MGRiNC0yN2E1LTRiZDYtYTdiZS0xMWI5ZWQ3OTlkZDMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMzBUMDkyMTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWUwY2ZiYTk1MjAzOTA2YTg2MjFhZWEzN2EwZDlmZmI5N2M3Y2MxZDgwOGM3MTU0YmM1OTVhNTdkNGUyYmQ2NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.I3geix3WiYWsEBWrrYHN8SxIbRRxaxZtAmnmjlcTv8k"> <img alt="Screenshot 2024-10-29 at 2 03 39 PM" width="1209" src="https://private-user-images.githubusercontent.com/1134620/381306382-1a3a490c-4441-4570-be18-479eeed03d18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyODA0MTQsIm5iZiI6MTczMDI4MDExNCwicGF0aCI6Ii8xMTM0NjIwLzM4MTMwNjM4Mi0xYTNhNDkwYy00NDQxLTQ1NzAtYmUxOC00NzllZWVkMDNkMTgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMzBUMDkyMTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjE3OTk1YWJiYjczOWQ1NDI3ODNlNjBhMGNjNTk2MWZjYjE2N2M4ODU2ZTI4NTZjZjc2ZmYyOThjMDdhOThmYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.zux5vPLoyu5G864jmabdO7A2UseXo7EVVKOVjaHu7u8">
> Reply with 🎉 to vote for the `rebeccapurple` logo Reply with 🚀 to vote for the `blue` logo

If we can only choose between the blue or purple, i'll go with blue. How bad that the pink didn't make it...

**Comment by jbwharris** [comment]
I'm fond of the purple. Not that it necessarily gets paired with icons on the homescreen of your mobile device, but there are so many apps that have white on blue logos. Steering away from that makes sense if you can create differentiation of identity.

**Comment by dragontheory** [comment]
Is it too late?
![01_CSS](https://github.com/user-attachments/assets/edaeea04-8b2f-4a66-b970-c8ce4d5ec355)
![02_CSS](https://github.com/user-attachments/assets/9cd75065-b925-47a9-9907-b543052c12e2)
![03_CSS](https://github.com/user-attachments/assets/ad5207af-d433-4971-bbc1-2cdc2c443516)
![04_CSS](https://github.com/user-attachments/assets/0bd2de76-6ec8-43cd-bb98-2d2a3aa78bf2)
![05_CSS](https://github.com/user-attachments/assets/2bcb35a8-9e50-4827-b671-621b71977a04)
![06_CSS](https://github.com/user-attachments/assets/b9086fb3-9e0f-4cef-be27-ab09337f99f4)
![07_CSS](https://github.com/user-attachments/assets/9e8b56cf-679f-4dbf-94c0-9647816708ba)
![08_CSS](https://github.com/user-attachments/assets/f0430b0e-ca2e-499f-8419-8f3f6e38dd9c)

**Comment by mxdvl** [comment]
Love the `rebeccapurple` logo from @itsjavi & @argyleink and [especially the Codepen where the logo is drawn using CSS only](https://codepen.io/argyleink/pen/jOgZpmy). I did, however, have a small concern in that it relies on an external font, so [I accurately reproduced the CSS letter shapes in SVG](https://svelte.dev/playground/b95d70159b2f4d9e962fd0eb0c45c060?version=5.1.8) so that the future logo can ben entirely self-contained:

```html
<svg viewBox="0 0 300 300" fill="none" stroke="white" stroke-width="22">
  <path fill="rebeccapurple" stroke="none" d="
    M0,0 H250
    A 50 50 0 0 1 300 50
    V250
    A 50 50 0 0 1 250 300
    H50
    A 50 50 0 0 1 0 250
    Z" 
  />
  <g id="css" transform="translate(70 220)">
    <path class="C" transform="translate(32 0)" d="
      M 21,-24
      C 21,-33 18,-48 0,-48
      C -18,-48 -21,-33 -21,-24
      V 24
      C -21,28 -18,48 0,48
      C 18,48 21,28  21,24
    " />
    <path class="S" transform="translate(104 0)" d="M 21,-24
      C 21,-33 18,-48 0,-48
      C -18,-48 -21,-33 -21,-24
      C -21,-14 -15,-6.25 0,0
      C 15,6.25 21,14 21,24
      C 21,28 18,48 0,48
      C -18,48 -21,28 -21,24"
    />
    <path class="S" transform="translate(176 0)" d="M 21,-24
      C 21,-33 18,-48 0,-48
      C -18,-48 -21,-33 -21,-24
      C -21,-14 -15,-6.25 0,0
      C 15,6.25 21,14 21,24
      C 21,28 18,48 0,48
      C -18,48 -21,28 -21,24"
    />
  </g>
</svg>
```

<img width="755" alt="screenshot comparison of CSS logo" src="https://github.com/user-attachments/assets/fb107eb3-c3ee-4dae-99a3-8f2f2effece4">


Feel free to reuse any parts of this if relevant!

**Comment by plutonium-239** [comment]
Wasm already has a purplish colour, with `rebeccapurple` the "with friends" becomes a bit boring. I really loved the pink suggestions, wish that was presented as an option.

**Comment by itsjavi** [comment]
thank you @mxdvl 

I think the final SVG will have the font as curves, or we can embed the font in CSS with a reduced charset to make it smaller

**Comment by argyleink** [comment]
> embed the font in CSS with a reduced charset to make it smaller

we were just talking about that this morning! as an alt route to making it easy to use this. we'll definitely have options for folks, with different variants and implementations so you can embed it easy 👍🏻

**Comment by SachaG** [comment]
I like the purple! 🎉 That being said I'm a bit sad that the CSS logo needs to be dragged down to the boring "acronym in a box" level of the other ones… Calling them "logos" at all is already pretty generous!

**Comment by argyleink** [comment]
the winner by popular vote: `rebeccapurple`

![Screenshot 2024-11-12 at 12 46 23 PM](https://github.com/user-attachments/assets/735eb5eb-3161-4b8c-b843-27e8e28d8431)

The final version (and new place for discussion) in this shiny new repo https://github.com/CSS-Next/logo.css

**Comment by ryanleichty** [comment]
The rounded corners supposedly nodding to tooltips gets totally lost on me. They feel dated already imo—I would say ditch them and go for continuity since that already seems to be a goal.

**Comment by LitoMore** [comment]
Are those

**Comment by LitoMore** [comment]
Hi @argyleink. Are those HTML, JS, and WASM logos decided among the CSS logo?

Can I use those new designs on [Simple Icon](https://github.com/simple-icons/simple-icons)?

**Comment by Que-tin** [comment]
> Hi @argyleink. Are those HTML, JS, and WASM logos decided among the CSS logo?
> 
> Can I use those new designs on [Simple Icons](https://github.com/simple-icons/simple-icons)?

Most of these logos have been around for quite some time and have just been used as a reference, besides HTML.

**WebAssembly**:
https://github.com/carlosbaraza/web-assembly-logo

**Typescript** looks slightly different, the one used as reference was the Community logo it was adapted and adjusted by Microsoft:
https://www.typescriptlang.org/branding/

The **JavaScript** logo is widely used by the community but as far as our research goes never was crowned as official logo. 

The **HTML** one is just an example on how a new one could look like, this one doesn't exist in the way it is represented.

TL;DR You can use the WebAssembly and Typescipt one from the official sources

**Comment by iamdtms** [comment]
- font kerning issue fixed (by @yuanchuan)
- inner stoke added
- not a drupal logo: border-radius realigned 

<img width="490" alt="688336ef5ede6afb" src="https://github.com/user-attachments/assets/07bf9bbb-cfe2-46e8-a7f0-f93593700ef5">

**Comment by ondrejkonec** [comment]
@itsjavi Hey Javi, it's possible to have your e-mail? I'm asking, because I wrote article about how you guys created new logo and would be great to have some original images if possible. Thanks.

**Comment by ondrejkonec** [comment]
CSS has a new logo and since there is also a beautiful story behind it, I wanted to share it in writing.

I called it [Kind of Rebeccapurple](https://www.ondrejkonecny.com/blog/kind-of-rebeccapurple/)

Thanks @Que-tin @una @argyleink
